action.yaml

```
@@ -55,6 +61,7 @@ runs:
         PR_BRANCH: ${{ github.head_ref }}
         VERBOSE: ${{ inputs.verbose }}
         WORKSPACE_PATH: ${{ steps.prerequisites.outputs.workspace_path }}
+        BAZEL_STARTUP_OPTIONS: ${{ inputs.bazel-start-options }}
```

Should this be

```suggestion
        BAZEL_STARTUP_OPTIONS: ${{ inputs.bazel-startup-options }}
```

?